### DISCUSSION TITLE
MSI Support

### DISCUSSION THREAD

**Discussion by chrpai** [ironfede/openmcdf#276] [General]
I'm looking at creating an SDK and tools using .net 8/9 with the help of this library.  I read through issue #11 and a few linked pages and was wondering what the current state of affairs is since that was all from 2017.

I'm open to contributing as MSI and dotnet is in my wheelhouse.

**Comment by Numpsy** [comment]
> what the current state of affairs is

You can open MSI files with the v3 structured storage explorer and view the streams, but aside from the known SummaryInformation stream it just operates on the binary data.

![image](https://github.com/user-attachments/assets/3513e5c5-2783-4e35-8ebd-6fa7ff686123)

**Comment by jeremy-visionaid** [comment]
Yeah, as @Numpsy said, there's no special handling for MSI. But if you wanted to contribute new library/module (e.g. OpenMcdf.Msi) then you are certainly welcome to do so!
